.pre-commit-config.yaml

```diff
@@ -18,20 +18,20 @@ repos:
       - id: black
         types: [python]
         additional_dependencies: ['click==8.0.4']
-        args: ["--extend-exclude", "topostats/plotting.py", "topostats", "tests", "*.ipynb"]
+        args: ["--extend-exclude", "topostats/plotting.py"]
 
   - repo: https://github.com/pycqa/flake8.git
     rev: 5.0.4
     hooks:
       - id: flake8
-        args: ["--extend-exclude=topostats/_version.py", "topostats", "tests", "*.ipynb"]
+        args: ["--extend-exclude=topostats/_version.py"]
         additional_dependencies: [flake8-print]
         types: [python]
   - repo: local
     hooks:
       - id: pylint
-        args: ["--rcfile=.pylintrc", "--ignore-patterns=*_version.py", "topostats", "tests"]
+        args: ["--rcfile=.pylintrc"]
```

Comment:
Does this mean that `.ipynb` notebook files will be checked with pylint ?

Reply:
Ah, good spot. I didn't copy over the necessary change to `setup.cfg` from #398 which excludes `ipynb` from `pylint`, once that is merged it will sort things it.

It was because #398 was awaiting approval and I wanted to get this quick improvement to validation out for PR that I copied the changes over, need to be more attentive next time.